packages/foreman/foreman-installer/foreman-installer.spec

```diff
@@ -10,55 +11,92 @@ Epoch:      1
 Version:    3.6.0
 Release:    %{?prerelease:0.}%{release}%{?prerelease:.}%{?prerelease}%{?nightly}%{?dist}
 Summary:    Puppet-based installer for The Foreman
-Group:      Applications/System
 License:    GPLv3+ and ASL 2.0
 URL:        https://theforeman.org
 Source0:    https://downloads.theforeman.org/%{name}/%{name}-%{version}%{?prerelease:-}%{?prerelease}.tar.bz2
 
 BuildArch:  noarch
 
-Requires:   curl
-Requires:   hostname
-Requires:   puppet-agent >= 6.15.0
-Requires:   %{?scl_prefix}rubygem(kafo) >= 6.5.0
-Requires:   %{?scl_prefix}rubygem(kafo) < 7.0.0
-Requires:   %{?scl_prefix_ruby}ruby(release)
+Requires: %{name}-scenario-foreman = %{epoch}:%{version}-%{release}
```

Comment:
I was not expecting this part, that foreman-installer main package would have a requires on a scenario

Reply:
I still debated this in my head. I first wrote it here for compatibility or upgrades. Initially when I wrote this PR I really wanted to create a pulp installer and the split part was mostly to have a decent base to work on.

Perhaps we should not have `foreman-installer` as a real package anymore at all, just make all the scenarios `Provide: foreman-installer = %{version}`?